docs/scenario-dashboard.md

```diff
@@ -0,0 +1,244 @@
+# Batch Scenario Manager
+
+monoDrive's Batch Scenario Manager enables easy batch processing of AV testing jobs. 
+The Batch Scenario Manager provides the client with visuals of custom configurations 
+and result viewing for better testing. 
+The application is designed to step through each configuration with pre-built and customizable 
+properties. Once all configurations are made, deploy and run every combination of the properties 
+selected in the simulation. Results show the status of the test after it has run, and keeps 
+different batches organized for review. 
```

Comment:
can we take a pass at rephrasing this first paragraph? It's currently kind of a tough read